vmm/src/config.rs

```diff
@@ -1247,8 +1247,8 @@ pub struct NetConfig {
     pub mac: MacAddr,
     #[serde(default)]
     pub host_mac: Option<MacAddr>,
-    #[serde(default = "default_netconfig_mtu")]
```

Comment:
Don't forget to remove the default value from the openapi yaml file.

Reply:
Thanks!